### ISSUE TITLE
Is it possible to not show a popup if the mouse cursor didn't move after pushing the Ctrl/Cmd key?

### ISSUE THREAD

**Issue by tadashi-aikawa** [nothingislost/obsidian-hover-editor#106]
Is it possible to not show a popup if the mouse cursor didn't move after pushing the `Ctrl/Cmd` key?

I often use a `Ctrl` key during viewing/editing a file (ex: `Ctrl + Left`, `Ctrl + Right`). When the mouse cursor is on the internal link or images, it shows a popup unintentionally and hides contents.

![2022-04-17](https://user-images.githubusercontent.com/9500018/163681881-f2da5c0a-362e-49d5-adfc-57e354f6f39c.gif)

Kind regards.

**Comment by nothingislost** [comment]
Hmm, most of the time when I'm triggering popovers, I move my mouse over the link and then press ctrl to initiate the popover without moving the mouse further... so basically the opposite of your use case :)

Perhaps I could cancel the hover preview if any additional key (other than cmd/ctrl) is pressed within a certain period of time

**Comment by HEmile** [comment]
I have to agree this is a very common problem for me too, since i have so many hotkeys mapped to cmd

**Comment by DryIce1** [comment]
I have been generally frustrated at times by inappropriate pop ups of hover windows when I didn't need them. 

But today I had a more specific interruption. I wanted to open links from the backlinks pane into new windows (ie CTRL click). But when I hover over the link, it pops up and then the backlink pane changes, and now a different link is under the mouse so a another hover for a different note pops up. And then it is a wild ride as it keeps cycling hovers back and forth from the backlinks pane. And in the end I found that I couldn't open the notes I want from the original note's backlinks pane.

![image](https://user-images.githubusercontent.com/59501376/167159260-7f80cff4-7479-4444-aad5-c6c37735ae32.png)
_(above; hover over the top linked mentions note on the backlinks page, and it will cycle through a few notes at a rapid pace)_

Perhaps this would be solved if there was a less disruptive trigger key? Is it possible to change the trigger key from CTRL / CMD to something else?

**Comment by huyz** [comment]
Oh so that's what was happening!  I had an impossible time renaming a file in the File Explorer because I can't select the full name with `cmd+A` or go to the end of the filename with `cmd+right` or paste with `cmd+V`.

Hmm yeah, this is becoming a show-stopper for me. I need to be able to rename files and because I'm using the Front Matter Title plugin, the File Explorer is the only way.

**Comment by ovanbiervliet** [comment]
Came here to find a way to disable the Cmd trigger. Without this setting, the plugin is currently unusable to me.

**Comment by pjeby** [comment]
If you want to disable the Cmd trigger, go to Obsidian's settings, and then the "Page Preview" **core plugin** (not community).  Turn off the toggles for the areas where you don't want cmd active.  Instead, the popup will happen *without* using cmd, so you will know immediately if you accidentally park your mouse on a link, instead of having it pop up later.

**Comment by ovanbiervliet** [comment]
@pjeby I figured it out after disabling the Hover editor and still seeing the popups. Glad I can now enable the hover editor again 👌
